allure-testng/src/test/java/io/qameta/allure/testng/AllureTestNgTest.java

```diff
@@ -1431,6 +1452,7 @@ public void idAssignToOtherTest() {
     @Test
     @AllureFeatures.Filtration
     public void skippedTest() {
+        System.setProperty(ALLURE_REPORT_DISABLED_TESTS, "true");
```

Comment:
make sure you cleanup the system property. I'm aware of `SetSystemProperty` JUnit5 extension, but not sure how it supposed to be done in TestNG.

Maybe we can simply provide config options directly to config POJO and test AllureTestNgConfig initialisation in separate test

Reply:
Done, thanks. Please, review.